packages/edit-post/src/components/secondary-sidebar/list-view-sidebar.js

```diff
@@ -43,10 +57,49 @@ export default function ListViewSidebar() {
 			onKeyDown={ closeOnEscape }
 		>
 			<div
-				className="edit-post-editor__list-view-panel-header"
+				className="edit-post-editor__list-view-panel-header components-panel__header edit-post-sidebar__panel-tabs"
 				ref={ headerFocusReturnRef }
 			>
-				<strong id={ labelId }>{ __( 'List View' ) }</strong>
+				<ul>
+					<li>
+						<Button
+							onClick={ () => {
+								setTab( 'list-view' );
+							} }
+							className={ classnames(
+								'edit-post-sidebar__panel-tab',
+								{ 'is-active': tab === 'list-view' }
+							) }
+							data-label={ __( 'List View' ) }
+							aria-label={
+								tab === 'list-view'
+									? __( 'List View (selected)' )
+									: __( 'List View' )
+							}
+						>
+							{ __( 'List View' ) }
+						</Button>
+					</li>
+					<li>
+						<Button
+							onClick={ () => {
+								setTab( 'outline' );
+							} }
+							className={ classnames(
+								'edit-post-sidebar__panel-tab',
+								{ 'is-active': tab === 'outline' }
+							) }
+							data-label={ __( 'Outline' ) }
+							aria-label={
```

Comment:
Changing `aria-label` text based on selected state works but is not recommended. Please use `aria-current="true/false"` instead. You can add it the same way you add the class.

Reply:
The other option is to render a tab control here instead of buttons. That should work just as well if not better for this use-case.

Reply:
Updated to use aria-current 👍

Reply:
As @alexstine suggested, an ARIA tabbed interfaced would be better. Will report in a new issue. Also, the initial focus skips the 'tabs'.

Reply:
> Will report in a new issue.

When doing so, could you mention https://github.com/WordPress/gutenberg/issues/45390 being a dependency? From what I understand, until manual tab activation is added to `TabPanel`, we won't be able to use it here

Reply:
Sure, thanks. If the problem is the initial focus, I think we should move `focusOnMountRef` to the `ListViewSidebar` container rather than set it on the list view container. That would avoid the focus to skip the tabs, regardless whether they're manually or automatically activated.

Reply:
@afercia That is not an option because the `ListView()` is rendered as an `role="application"`. Focus needs to be placed there on mount even if we risk skipping the tabs. There is a long outstanding NVDA bug about this.

https://github.com/nvaccess/nvda/issues/12896

Reply:
Thanks @alexstine can you please mention this on the new issue #45899 ?
Will read about the NVDA bug as soon as I have some time.